rust-sdk/whirlpool/src/tests/program.rs

```diff
@@ -60,14 +75,165 @@ pub async fn setup_whirlpool(
     Ok(whirlpool)
 }
 
-pub async fn setup_position(_whirlpool: Pubkey) -> Result<Pubkey, Box<dyn Error>> {
-    todo!()
+pub async fn setup_position(whirlpool: Pubkey) -> Result<Pubkey, Box<dyn Error>> {
+    let ctx = RpcContext::new().await;
+
+    // Use token utility functions
+    let position_mint = setup_mint_with_decimals(&ctx, 0).await?;
```

Comment:
You don't have to setup mint and ata when opening a position. The OpenPosition instruction will do that